py/htm/examples/mnist.py

```diff
@@ -94,20 +80,17 @@ def load_images(file_name):
 
 
 def main(parameters=default_parameters, argv=None, verbose=True):
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--data_dir', type=str,
-        default = os.path.join( os.path.dirname(__file__), '..', '..', '..', 'build', 'ThirdParty', 'mnist_data', 'mnist-src'))
-    args = parser.parse_args(args = argv)
 
     # Load data.
-    train_labels, train_images, test_labels, test_images = load_mnist(args.data_dir)
+    train_labels, train_images, test_labels, test_images = load_ds('mnist_784', 10000, shape=[28,28]) # HTM: ~95.6%
+    #train_labels, train_images, test_labels, test_images = load_ds('Fashion-MNIST', 10000, shape=[28,28]) # HTM baseline: ~83%
```

Comment:
it's now this simple to play with a new image classification dataset! 
Even the results for fashionMNIST are not that bad (83% without touching anything)

Reply:
..disadvantage is that one hast to be online, and that the dataset is re-downloaded each time. Can we live with that?